ci-operator/step-registry/ipi/conf/capability/additonalcaps/ipi-conf-capability-additonalcaps-commands.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ "${BASELINE_CAPABILITY_SET}" == "" ]]; then
+  echo "This step requires BASELINE_CAPABILITY_SET to be set!"
+  exit 1
+fi
+
```

Comment:
Per https://github.com/openshift/release/pull/42999/files#diff-d52f5aefd0e6f5877090fb4dee62ee1fe2666bd9999a1e828913e091ea65f701R139, seem like has higher priority than this step, so can we add one more pre-check to avoid both are enabled.

Reply:
updated.

Reply:
Shall we also update the ref yaml file to receive `ADDITIONAL_ENABLED_CAPABILITIES` ?